decidim-initiatives/app/views/decidim/initiatives/admin/initiatives/index.html.erb

```diff
@@ -68,7 +68,7 @@
 
               <% if allowed_to? :read, :initiative, initiative: initiative %>
                 <%= icon_link_to "print",
-                                 decidim_admin_initiatives.initiative_path(initiative.to_param),
+                                 decidim_initiatives.print_initiative_path(initiative),
```

Comment:
I just have one suggestion here: could we add `target="_blank"` here because it takes the user out of the admin panel which is a weird experience.

In fact, many applications do the print versions so that it opens in a small external window which is then printed. This would improve the user experience but I understand it's not in the scope of this PR, so for now let's just add the `target="_blank"`.

Reply:
Good idea, I've added it.

Regarding the print layout, I agree. I'll do that in another PR, as for me that'd be a new feature instead of a bug fix.